src/gennylib/src/event.cpp

```diff
@@ -0,0 +1,7 @@
+#include <metrics/v2/event.hpp>
+
+namespace genny::metrics::internals::v2 {
+
+std::unique_ptr<poplar::PoplarEventCollector::StubInterface> CollectorStubInterface::_stub(nullptr);
+
```

Comment:
Note to self - come back here.

1. `event.cpp` is a strange name for this imho.
2. Can this instead be done inside the impl's cpp?
3. if nothing else plz comment what this is

Reply:
(still not convinced we need this but C++ static initialization is annoying (but getting better in C++20 I've heard). I think this would be much clearer with a comment, perhaps a different name for the cpp, and using assignment syntax `StubInterface::_stub = nullptr` rather than a [vexing parse](https://en.wikipedia.org/wiki/Most_vexing_parse).